packages/terra-form-radio/tests/wdio/radio-spec.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-expressions */
+/* global browser, Terra, before */
+
```

Comment:
You'll want to specify you're viewport size(s) at the top like 
```const viewports = Terra.viewports('tiny', 'medium');```
And then pass them into the your test steps as so:
```
    Terra.should.beAccessible({ viewports });
    Terra.should.matchScreenshot({ viewports });
```
You could also loop the viewport sizes, but since these are static components this would be the better option.